bin/run_multiple_panels.py

```diff
@@ -0,0 +1,443 @@
+
+        # Input parameters
+        'hpxml_file': os.path.join(input_path, 'Input Files', 'in.xml'),
+        'schedule_input_file': os.path.join(input_path, 'Input Files', 'schedules.csv'),
+        'weather_path': weather_path,
+    
+        # Output parameters
+        'output_path': input_path,
+        'output_to_parquet': False,              # saves time series files as parquet files (False saves as csv files)
+        'verbosity': 7,                         # verbosity of time series files (0-9)
+        
+        'seed': int(input_path[-3:]),
+        
+        'Equipment': {
+        },
+    }
+
+    # determine output path, default uses same as input path
+    if output_path is None:
+        output_path = input_path
+    os.makedirs(output_path, exist_ok=True)
+    
+    
+    # create OCHRE building based on der type
+    if der_type == 'ev':
+        dwelling = setup_ev_run(simulation_name, dwelling_args)
+    else: 
+        # baseline ResStock dwelling
+        dwelling = Dwelling(simulation_name,**dwelling_args)           
+                            
+    # run simulation
+    if sim_type == 'baseline':
+        # run without controls
+        dwelling.simulate()
+    
+    elif sim_type == 'circuit_sharing':
+        circuit_sharing_control(sim_type, dwelling, tech1, tech2, output_path)
+    
+    elif sim_type == 'circuit_pausing':
+        circuit_pausing_control(sim_type, input_path, dwelling, tech1, output_path)
+    
+    
+def setup_ev_run(simulation_name, dwelling_args):
+
+
+    equipment_args = {
+        'Electric Vehicle':{
+            # Equipment parameters
+            'vehicle_type': 'BEV',
+            'charging_level': 'Level 2',
+            "capacity": 57.5,
+        }
+    }
+
+    # Initialize equipment
+    # equipment = ElectricVehicle(**equipment_args)
+
+    dwelling_args['Equipment']=equipment_args
+    # print(dwelling_args)
+    dwelling = Dwelling(simulation_name, **dwelling_args)
+
+
+    return dwelling
+
+
+def circuit_sharing_control(sim_type, dwelling, tech1, tech2, output_path):   
+    
+    # run simulation with circuit sharing controls
+    times = pd.date_range(dwelling.start_time, dwelling.start_time + dwelling.duration, freq=dwelling.time_res,
+                          inclusive='left')
+
+    control_signal = None
+    
+    if tech2 == 'Clothes Dryer':
+        # initialize an empty list for cycle recording
+        pipeline = [] # for storing cycles waiting to be rescheduled
+        n_delay = 0 # number of cycles waiting in the pipeline
+        N = 0 # total number of delayed cycles
+        t_delay = [] # for storing delayed timesteps of each delayed cycle
+        n_pop = 0 # index for tracking number of cycles poped
+    
+    for t in times:
+        # print(t, dwelling.current_time)
+        assert dwelling.current_time == t
+        house_status = dwelling.update(control_signal=control_signal)
+    
+        # get primary load power
+        if tech1 in ['Strip Heat']:
```

Comment:
We excluded this, but good to have this capability. I think it's much harder to do in practice than it is to do in simulation.